scripts/climate

```diff
@@ -49,6 +49,7 @@ my $base = abs_path(getcwd);
 
 # The threshold is the percentage in which we consider that the given package
 # is covered enough by tests. It defaults to 90.0%
+# TODO: change default back to 90% once coverage is up again
```

Comment:
Good catch! This is a leftover from a previous version of the patch. As some tests didn't work, the coverage dropped below 90%. Only after changing this I saw which tests broke, that's also why I changed the execution order in the Makefile. I'll fix it.